importApp/src/main/java/com/commercetools/dataimport/orders/OrderImportItemReader.java

```diff
@@ -0,0 +1,127 @@
+package com.commercetools.dataimport.orders;
+
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.orders.*;
+import io.sphere.sdk.products.Price;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.ItemStream;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.batch.item.file.FlatFileItemReader;
+import org.springframework.batch.item.file.mapping.BeanWrapperFieldSetMapper;
+import org.springframework.batch.item.file.mapping.DefaultLineMapper;
+import org.springframework.batch.item.file.transform.DelimitedLineTokenizer;
+import org.springframework.batch.item.support.SingleItemPeekableItemReader;
+import org.springframework.core.io.Resource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+
+public class OrderImportItemReader implements ItemReader<OrderImportDraft>, ItemStream {
+
+    public static final String[] ORDER_CSV_HEADER_NAMES = new String[]{"customerEmail", "orderNumber", "lineitems.variant.sku", "lineitems.price", "lineitems.quantity", "totalPrice"};
+
+    private final SingleItemPeekableItemReader<OrderCsvLineValue> singleItemPeekableItemReader;
+
+    private final FlatFileItemReader flatFileItemReader;
+
+
+    public OrderImportItemReader() {
+
+        flatFileItemReader = new FlatFileItemReader<>();
+        flatFileItemReader.setLineMapper(new DefaultLineMapper<OrderCsvLineValue>() {{
+            setLineTokenizer(new DelimitedLineTokenizer() {{
+                setNames(ORDER_CSV_HEADER_NAMES);
+
+            }});
+            setFieldSetMapper(new BeanWrapperFieldSetMapper<OrderCsvLineValue>() {{
+                setTargetType(OrderCsvLineValue.class);
+            }});
+        }});
+        flatFileItemReader.setLinesToSkip(1);
+
+        singleItemPeekableItemReader = new SingleItemPeekableItemReader<>();
+        singleItemPeekableItemReader.setDelegate(flatFileItemReader);
+
+    }
+
+    public void setResource(Resource resource) {
+        this.flatFileItemReader.setResource(resource);
+    }
+
+    @Override
+    public void open(ExecutionContext executionContext) throws ItemStreamException {
+        singleItemPeekableItemReader.open(executionContext);
+    }
+
+    @Override
+    public void update(ExecutionContext executionContext) throws ItemStreamException {
+        singleItemPeekableItemReader.update(executionContext);
+    }
+
+    @Override
+    public void close() throws ItemStreamException {
+        singleItemPeekableItemReader.close();
+    }
+
+    @Override
+    public OrderImportDraft read() throws Exception {
+
+        OrderCsvLineValue currentLine = this.singleItemPeekableItemReader.read();
```

Comment:
Why aren't we using `this.singleItemPeekableItemReader.peek()` here? If use it here, we can get rid of the special handling of the first item below. Or am I wrong?